frontend/TestRun/Jenkins/SCTParameterWizard.svelte

```diff
@@ -266,7 +266,8 @@
                     validationHelpText: "AMI Id should be in the format of \"ami-hexid\"",
                     validate: function (params) {
                         const AMI_RE = /^ami-([abcdef\d]{4,17})$/;
-                        return !!params[this.internalName] && AMI_RE.test(params[this.internalName]);
+                        const AMI_FUZZY_RE = /^scylla-\d{4}\.\d\.\d(-dev)?(-[\w\d_]*)(-\d{4}-\d{2}-\d{2})T(\d{2}-?){3}/;
```

Comment:
I think we could be less strict and test only if it starts with scylla-\d{4}
Better not to add this maintenance burden when someone changes this pattern